bundle-ocata-s390x.yaml

```diff
@@ -190,7 +190,7 @@ services:
     num_units: 1
     options:
       worker-multiplier: 0.5
-      bridge-mappings: physnet1:br-ex
+      ext-port: eth1
```

Comment:
We should keep the new-style network declaration (physnet).  Is there a bug causing the need to change this to the old-style network declaration?